pep-0704.rst

```diff
@@ -0,0 +1,170 @@
+PEP: 704
+Title: Require virtual environments by default for package installers
+Author: Pradyun Gedam <pradyunsg@gmail.com>
+Sponsor: Brett Cannon <brett@python.org>
+PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
+Discussions-To: https://discuss.python.org/t/22846
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Content-Type: text/x-rst
+Created: 16-Jan-2023
+Post-History: `16-Jan-2023 <https://discuss.python.org/t/22846>`__
+
+
+Abstract
+========
+
+This PEP recommends that package installers like ``pip``  require a virtual environment by default, on Python 3.13+.
+
+
+Motivation
+==========
+
+Python virtual environments are an essential part of the development workflow for Python. However, they require extra effort since they are an opt-in feature, and requires users to either:
+
+- take explicit steps to activate/deactivate a virtual environment
+- use :samp:`{<path-to-venv>}/{<bin-path>}/{<executable>}` to run files
+
+For new users, things will seemingly work correctly when virtual environments are not used until they don't. Further, activating a virtual environment uses slightly different syntax and mechanisms on different platforms. This complicates the introduction to virtual environments since now information and context about how/why they are useful needs to explained to justify adding additional steps to the workflow.
```

Comment:
```suggestion
For new users, things will seemingly work correctly when virtual environments are not used -— until they don't. Further, activating a virtual environment uses slightly different syntax and mechanisms on different platforms. This complicates the introduction to virtual environments, since now information and context about how/why they are useful needs to explained to justify adding additional steps to the workflow.
```